### PR TITLE
Update ES upgrade guide with prerequisites

### PR DESCRIPTION
**Pull request by roborourke**
Important step for hassle free upgrades.

### CODE REVIEW

File: other-docs/guides/updating-elasticsearch/README.md [hunk 1]
```diff
@@ -12,6 +12,19 @@ Altis v5+ is compatible with the following Elasticsearch versions:
...
 For all Altis environments created before 2021-09-01, the default Elasticsearch version is 6.3. For all environments created after this date, the default version is 7.10.
 
+## Pre-Requisites
+
+There several things you must ensure have been done before updating your Elasticsearch version locally _or_ when requesting an update for a cloud environment.
+
+These requirements will help to avoid any errors and delays.
+
+1. Ensure all Altis modules are fully up to date by running `composer update "altis/*" --with-all-dependencies`
+1. If the upgrade is for a cloud environment ensure the updated `composer.lock` has been committed and deployed
```
Comment (kadamwhite) [inline other-docs/guides/updating-elasticsearch/README.md:22]:
```suggestion
1. If the upgrade is for a cloud environment, ensure the updated `composer.lock` has been committed and deployed
```

File: other-docs/guides/updating-elasticsearch/README.md [hunk 2]
```diff
@@ -12,6 +12,19 @@ Altis v5+ is compatible with the following Elasticsearch versions:
...
+1. If any patch updates were downloaded, in particular for the Cloud or Search modules you should reindex your content using one of the following commands:
+   - Local Chassis: `composer chassis exec -- wp elasticpress index --setup --network-wide`
+   - Local Server: `composer server cli -- elasticpress index --setup --network-wide`
```
Comment (kadamwhite) [inline other-docs/guides/updating-elasticsearch/README.md:25]: Let's lead with Server, it's the recommended approach of late

File: other-docs/guides/updating-elasticsearch/README.md [hunk 3]
```diff
@@ -12,6 +12,19 @@ Altis v5+ is compatible with the following Elasticsearch versions:
...
+   - Cloud: `wp elasticpress index --setup --network-wide`
+
```
Comment (kadamwhite) [inline other-docs/guides/updating-elasticsearch/README.md:27]: Alright, I've done all that. Now what? We continue in the next section, but I'd like to see this one end with something like,

> After completing the above steps to ensure your environment is ready for the Elasticsearch upgrade, you can then proceed to the steps below.

File: other-docs/guides/updating-elasticsearch/README.md [hunk 4]
```diff
@@ -12,6 +12,19 @@ Altis v5+ is compatible with the following Elasticsearch versions:
 
 For all Altis environments created before 2021-09-01, the default Elasticsearch version is 6.3. For all environments created after this date, the default version is 7.10.
 
+## Pre-Requisites
```
Comment (kadamwhite) [inline other-docs/guides/updating-elasticsearch/README.md:15]:
```suggestion
## Prerequisites
```

File: other-docs/guides/updating-elasticsearch/README.md (hunk 3 above)
Comment (roborourke) [reply other-docs/guides/updating-elasticsearch/README.md:27]:
```suggestion

After completing the above steps to ensure your environment is ready for the Elasticsearch upgrade, you can then proceed to the steps below.
```

File: other-docs/guides/updating-elasticsearch/README.md (hunk 2 above)
Comment (roborourke) [inline other-docs/guides/updating-elasticsearch/README.md:25]:
```suggestion
   - Local Server: `composer server cli -- elasticpress index --setup --network-wide`
   - Local Chassis: `composer chassis exec -- wp elasticpress index --setup --network-wide`
```

File: other-docs/guides/updating-elasticsearch/README.md [hunk 5]
```diff
@@ -12,6 +12,20 @@ Altis v5+ is compatible with the following Elasticsearch versions:
...
+These requirements will help to avoid any errors and delays.
+
+1. Ensure all Altis modules are fully up to date by running `composer update "altis/*" --with-all-dependencies`
+1. If the upgrade is for a cloud environment, ensure the updated `composer.lock` has been committed and deployed
+1. If any patch updates were downloaded, in particular for the Cloud or Search modules you should reindex your content using one of the following commands:
+   - Local Server: `composer server cli -- elasticpress index --setup --network-wide`
+   - Local Chassis: `composer chassis exec -- wp elasticpress index --setup --network-wide`
+   - Cloud: `wp elasticpress index --setup --network-wide`
+
+After completing the above steps to ensure your environment is ready for the Elasticsearch upgrade, you can then proceed to the steps below.
```
Comment (roborourke) [inline other-docs/guides/updating-elasticsearch/README.md:28]:
```suggestion
After completing the above steps to ensure your environment is ready for the Elasticsearch upgrade, you can then proceed to the steps below.

```

**Comment by roborourke** [comment]
Thanks @kadamwhite, should be good to go

**Review by kadamwhite — APPROVED** [review]
:shipit:

**Pull request merged**